src/Components/CreateImageWizard/steps/Oscap/index.tsx

```diff
@@ -117,6 +127,15 @@ const OscapContent = () => {
         versions. This will automatically help monitor the adherence of your
         registered RHEL systems to a selected policy or profile.
       </Content>
+      <FormGroup>
+        <Checkbox
+          id='fips-enabled-checkbox'
+          label='Enable FIPS mode'
+          isChecked={fips.enabled}
+          onChange={(_event, checked) => handleFipsToggle(checked)}
+          description='Enable FIPS 140-2 compliant cryptographic algorithms. This setting will be applied at build time and will persist on boot.'
```

Comment:
Quick question, I'm not sure on the copy here. Or was this given to you in the mocks?

Reply:
@kingsleyzissou I asked about it in the channel - https://redhat-internal.slack.com/archives/C05LNKHBCDA/p1754410094945309?thread_ts=1754408956.763919&cid=C05LNKHBCDA
if you have any suggestion here, jsut tell me